migrations/app/secure/20230808155723_add_ns_mayport.up.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE addresses SET postal_code = '32233' WHERE id = '430c50b4-cc87-42a6-9921-2010aa6d4ed3';
```

Comment:
I think having a comment here makes it easier for folks looking for where this "secure migration" run.

```suggestion
--- This migration is only expected to run in the STG environment. The other environments have a blank file.

UPDATE addresses SET postal_code = '32233' WHERE id = '430c50b4-cc87-42a6-9921-2010aa6d4ed3';
```

Reply:
Feel free to edit this comment in anyway you want. It might make sense to also call out that the information in this SQL file is what will be running in STG.